open-metadata-implementation/repository-services/repository-services-apis/src/main/java/org/odpi/openmetadata/repositoryservices/ffdc/OMRSAuditCode.java

```diff
@@ -980,6 +980,14 @@
                             "is not current registered and so the hope is that by retrying, the entity is returned on a subsequent attempt.",
                     "If this message occurs frequently then seek to improve the availability of the cohort members."),
 
+    SAVE_PROXY_FAIL("OMRS-AUDIT-8009",
+            OMRSAuditLogRecordSeverity.INFO,
+            "Fail to save proxy from {0} with guid {1} for user {2}, because: {3}",
```

Comment:
Add exception name as well as exception message.  Also the severity should be ERROR or EXCEPTION depending on whether you want the stack trace displayed on the console.